src/tool/subcommands/benchmark_cmd.rs

```diff
@@ -111,6 +124,36 @@ async fn benchmark_car_streaming(input: Vec<PathBuf>) -> Result<()> {
     Ok(())
 }
 
+// Concatenate a set of CAR files and measure how quickly we can stream the
+// blocks, while inspecting them. This a benchmark we could use for setting
+// realistic expectations in terms of DFS graph travels, for example.
+async fn benchmark_car_streaming_inspect(input: Vec<PathBuf>) -> Result<()> {
+    let mut sink = indicatif_sink("traversed");
+    let ipld_to_cid = |ipld| {
+        if let Ipld::Link(cid) = ipld {
+            return Some((cid, ()));
+        }
+        None
+    };
+
+    let mut s = Box::pin(
```

Comment:
why not pin to the stack?

Reply:
Just a copy-paste from the original bench without inspection. Good point though, I'm planning to perhaps unify the two benches going forward.